src/libraries/Directory.Build.targets

```diff
@@ -231,6 +232,9 @@
 
   <PropertyGroup>
     <ExcludeFromPackage Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', '$(NetCoreAppCurrent)')) and '$(ExcludeCurrentNetCoreAppFromPackage)' == 'true'">true</ExcludeFromPackage>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>.dll;.exe;.winmd;.json;.pri;.xml;</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <IncludeSymbols>true</IncludeSymbols>
+    <PackageVersion Condition="'$(StablizePackageVersion)' != 'true'">$(PackageVersion)-$(VersionSuffix)</PackageVersion>
```

Comment:
That property is already defined here: https://github.com/dotnet/runtime/blob/1df2bc9018846da1b1ba4ba38cf04273388c7c27/eng/packaging.props#L13. We probably don't want yet another different meaning of it as we already have four of it in the repo?

Reply:
This is transitory fix, First package version is consumed by the old infra and this one is consumed by the pack task.
We can merge them after we have ported all the projects to dotnet pack

Reply:
We should leverage the `PackageVersion` logic to arcade. I don't think we should be duplicating logic by checking `StabilizePackageVersion` and setting the `VersionSuffix` ourselves.

The way arcade does it is by appending `VersionSuffix` always if it is not empty, because it has a lot of calculations to decide whether we need a `VersionSuffix` or not:
https://github.com/dotnet/arcade/blob/99b9f805da070d301d76eac578a46dd75d9e5f7c/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L165

Reply:
I now better understand why it's uncertain that we want to use Arcade's default version calculation. Currently we hardcode the PackageVersion to $(MajorVersion).$(MinorVersion).0 and manually rev the PackageVersion for projects that are part of an incremental servicing release.

That said I tend towards let Arcade default the `PackageVersion` to `Version` which for the next servicing releases as an example is 5.0.5. Packages that incrementally service specify a `PackageVersion` anyway and if they don't they default to the current patch digit. I see advantages of that mechanism over the hardcoded `.0` patch version which would actually cause publishing to fail if no `PackageVersion` is specified (only talking about NuGet pack task without pkgproj and packageIndex infra).

Any concerns about that? cc @ericstj

Reply:
Talked with @ericstj about this offline and we agree that defaulting the PackageVersion to Version (SDK default behavior) is the right approach. This will mean that by default, i.e. for 5.0.5 servicing, the PackageVersion will be 5.0.5. As we incrementally service packages anyway, the `PatchVersion` will need to be set explicitly anyway (likely via an inferred property to also handle the required AssemblyVersion bump).

Reply:
I believe the conclusion was that  in servicing all libraries need to by default be frozen to Major.0.0 since we will not be shipping them by default.  This ensures that P2P references get the correct value for CSProjs that are being packed.  Then as we decide to ship a library it can declare the new servicing version as a single integer and we can use that to compute both the PackageVersion and AssemblyVersion (as appropriate).

Reply:
Oh gosh, I forgot about that part of the discussion, you are absolutely right.

I propose to remove the line above and only hardcode the `PackageVersion` (in Versions.props) during servicing to $(MajorVersion).$(MinorVersion).0. We should make sure that properties like `PackageVersion` that apply to every package in the repository are defined centrally (ie in Directory.Build.* files or Versions.props). Related: https://github.com/dotnet/runtime/issues/48229